relevanceai/__init__.py

```diff
@@ -8,3 +9,11 @@
     pass
 
 __version__ = "0.24.8"
+
+latest_version = requests.get("https://pypi.org/pypi/relevanceai/json").json()["info"][

```

Comment:
You need to put this in a try-except loop. If Pypi changes links, this will break unnecessarily

Reply:
Also do not use square bracket entries here. You need to use `get` method for dictionaries